.github/workflows/release.yaml

```
@@ -74,13 +74,16 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@master
+      - name: Set up Go 1.17
+        uses: actions/setup-go@v1
         with:
-          platforms: all
-      - name: Set up Docker Buildx
-        id: buildx
-        uses: docker/setup-buildx-action@master
+          go-version: 1.17.3
```

nit: 1.17.6 is latest
```suggestion
          go-version: 1.17.6
```

---

updated all steps to use 1.17.6; we can parameterize this as the release grows in complexity?